addons/mail/models/mail_channel.py

```diff
@@ -230,12 +230,16 @@ def _action_unfollow(self, partner):
         self.message_unsubscribe(partner.ids)
         if partner not in self.with_context(active_test=False).channel_partner_ids:
             return True
-        channel_info = self.channel_info('unsubscribe')[0]  # must be computed before leaving the channel (access rights)
+        channel_info = self.channel_info()[0]  # must be computed before leaving the channel (access rights)
         result = self.write({'channel_partner_ids': [Command.unlink(partner.id)]})
         # side effect of unsubscribe that wasn't taken into account because
         # channel_info is called before actually unpinning the channel
         channel_info['is_pinned'] = False
-        self.env['bus.bus'].sendone((self._cr.dbname, 'res.partner', partner.id), channel_info)
+        self.env['bus.bus'].sendone(
+            partner,
+            'mail.unfollow',
```

Comment:
The name of the original method is bad, we don't want to use the "follow" term for channel members.

For now I suggest `mail.channel_leave` or more generic (we will need it later) `mail.channel_remove_members`.

The payload in that case should be the channel id and the members (= partners for now) that left, probably nothing else and definitely not the full channel info.

And it should be sent to the whole channel, not just the leaving partner.